### ISSUE TITLE
Switch go builds from epel-testing to epel-stable 

### ISSUE THREAD

**Issue by alexeykazakov** [openshiftio/openshift.io#4618]
We need to switch all our go repos from using go from epel-testing to epel-stable

**Comment by alexeykazakov** [comment]
cc: @pbergene @sbose78 @sunilk747 @chmouel @ibuziuk

**Comment by alexeykazakov** [comment]
@corinnekrych absolutely. We are planning to investigate goa 1.4 move in our cluster service - https://github.com/fabric8-services/fabric8-cluster/issues/13
Cluster service is tiny and looks like a good candidate for the first try.
But this is not a blocker for using go 1.11 - https://github.com/fabric8-services/build-tool-detector/pull/28#issuecomment-445930809

**Comment by nurali-techie** [comment]
@sbose78 I see that f8-wit also would face same issue with http/proxy and reverseproxy which f8-auth faced in last week.

Issue: https://github.com/fabric8-services/fabric8-auth/issues/734
Fixed f8-common: https://github.com/fabric8-services/fabric8-common/blob/master/httpsupport/proxy.go#L165-L186
Need Fix f8-wit: https://github.com/fabric8-services/fabric8-wit/blob/master/rest/proxy/proxy.go#L124-L144

cc: @alexeykazakov

**Comment by alexeykazakov** [comment]
Right. This is why I’ve created a PR to update WIT a week ago - https://github.com/fabric8-services/fabric8-wit/pull/2377
Still waiting for review.
Just be aware that you are about to break proxy used in WIT without merging https://github.com/fabric8-services/fabric8-wit/pull/2377
